packages/rum-core/src/domain/rumEventsCollection/view/viewCollection.spec.ts

```diff
@@ -1,10 +1,11 @@
-import { Duration, RelativeTime, ServerDuration, TimeStamp } from '@datadog/browser-core'
+import { Duration, RelativeTime, ServerDuration, TimeStamp, Configuration } from '@datadog/browser-core'
 import { setup, TestSetupBuilder } from '../../../../test/specHelper'
 import { RumEventType, ViewLoadingType } from '../../../rawRumEvent.types'
 import { LifeCycleEventType } from '../../lifeCycle'
 import { ViewEvent } from './trackViews'
 import { startViewCollection } from './viewCollection'
 
+const configuration: Partial<Configuration> = { isEnabled: () => true }
```

Comment:
Same here, the configuration provided by the "setup builder" should be used instead